plugins/dell-esrt/dell-esrt.conf

```diff
@@ -2,7 +2,7 @@
 
 # this remote provides metadata shipped with the fwupd package
 Enabled=true
-Title=Core
+Title=Enable UEFI capsule updates on some systems
```

Comment:
I think you want the "Dell" word in here somewhere; if it's Dell specific it should probably give some hint in the title.